### PR TITLE
Add FXIOS-9218 Uncomment edit address strings to enable export

### PR DESCRIPTION
**Pull request by razvanlitianu**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-9218)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/20431)

## :bulb: Description

Add strings for address autofill phase 2.
## :pencil: Checklist
You have to check all boxes before merging
- [x] Filled in the above information (tickets numbers and description of your work)
- [x] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] ~~Wrote unit tests and/or ensured the tests suite is passing~~
- [ ] ~~When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)~~
- [ ] ~~If needed, I updated documentation / comments for complex code and public methods~~
- [ ] ~~If needed, added a backport comment (example `@Mergifyio backport release/v120`)~~

### CODE REVIEW

**Review by nbhasin2 — APPROVED** [review]

File: firefox-ios/Client/Frontend/Strings.swift [hunk 1]
```diff
@@ -282,219 +282,278 @@ extension String {
...
                 public static let AutofillAddressZip = MZLocalizedString(
-                    key: "", // "Addresses.EditAddress.AutofillAddressZip.v129"
+                    key: "Addresses.EditAddress.AutofillAddressZip.v129",
                     tableName: "EditAddress",
                     value: "ZIP Code",
                     comment: "Label for the ZIP code field, primarily used in the United States for mail sorting. Key for autofill to accurately complete addresses for shipping, billing, and service provision."
                 )
                 public static let AutofillAddressEircode = MZLocalizedString(
-                    key: "", // "Addresses.EditAddress.AutofillAddressEircode.v129"
+                    key: "Addresses.EditAddress.AutofillAddressEircode.v129",
```
Comment (nbhasin2) [inline firefox-ios/Client/Frontend/Strings.swift:508]:
```suggestion
                    key: "Addresses.EditAddress.AutofillAddressErrcode.v129",
```
Nit: string name fix

File: firefox-ios/Client/Frontend/Strings.swift [hunk 2]
```diff
@@ -282,219 +282,278 @@ extension String {
...
                 public static let AutofillAddressEmirate = MZLocalizedString(
-                    key: "", // "Addresses.EditAddress.AutofillAddressEmirate.v129"
+                    key: "Addresses.EditAddress.AutofillAddressEmirate.v129",
                     tableName: "EditAddress",
                     value: "Emirate",
                     comment: "Label for the emirate field, essential for addresses in the United Arab Emirates. Including emirate details ensures the autofill feature accurately represents user addresses."
                 )
                 public static let AutofillAddressOblast = MZLocalizedString(
-                    key: "", // "Addresses.EditAddress.AutofillAddressOblast.v129"
+                    key: "Addresses.EditAddress.AutofillAddressOblast.v129",
```
Comment (nbhasin2) [inline firefox-ios/Client/Frontend/Strings.swift:484]: Q. Whats 0blast? @razvanlitianu

Comment (nbhasin2) [reply firefox-ios/Client/Frontend/Strings.swift:484]: Ah Oblast 😅 kinda looked liked 0 blast

File: firefox-ios/Client/Frontend/Strings.swift (hunk 1 above)
Comment (razvanlitianu) [reply firefox-ios/Client/Frontend/Strings.swift:508]: hey, it's actually correct. Eircode is the national postcode system in Ireland.

File: firefox-ios/Client/Frontend/Strings.swift (hunk 2 above)
Comment (razvanlitianu) [reply firefox-ios/Client/Frontend/Strings.swift:484]: Yea, google says it's an administrative division of the constituent republics of Russia.

**Pull request merged**